src/Platform.Handlers/src/Xamarin.Platform.Handlers/Core/IProgress.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Xamarin.Forms;
+
+namespace Xamarin.Platform
+{
+	public interface IProgress : IView
+	{
+		double Progress { get; }
+		Color ProgressColor { get; }
```

Comment:
ProgressColor or maybe something to do with foreground?

Reply:
Do you mean use a Brush?. Have sense for me but probably not in this PR, after include Brushes support (P2-P3?).